### PR TITLE
Changed weight tests to include error message comparison.

### PR DESCRIPTION
**Pull request by JulietteDebray**
Test in grams pass. Will switch to new branch to continue writing service methods.

### CODE REVIEW

File: PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/service/EnvelopeServiceTest.java [hunk 1]
```diff
@@ -162,55 +164,59 @@ public void testEnvelopeTooLargeLengthInInches() {
...
-                "The envelope is too light");
+        Exception thrownException = assertThrows(ResponseStatusException.class, () ->
+                        envelopeService.calculatePostalRate(envelope));
+
+        assertTrue(thrownException.getMessage().contains("The envelope's weight is not within the allowed range."));
     }
 
     @Test
-    private void testEnvelopeTooHeavyInOunces() {
+    public void testEnvelopeTooHeavyInOunces() {
```
Comment (antoninguerre) [inline PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/service/EnvelopeServiceTest.java:195]: this shouldnt pass

Comment (JulietteDebray) [reply PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/service/EnvelopeServiceTest.java:195]: ya

**Comment by antoninguerre** [comment]
closes #25

**Review by antoninguerre — APPROVED** [review]
Looking good!

**Pull request merged**